### PR TITLE
Add a per-method doc comment with the API version

### PR DESCRIPTION
**Pull request by jhendrixMSFT**

### CODE REVIEW

**Comment by jhendrixMSFT** [comment]
Fixes https://github.com/Azure/azure-sdk-for-go/issues/17679

File: src/generator/operations.ts [hunk 1]
```diff
@@ -587,6 +587,17 @@ function emitPagerDefinition(op: Operation, imports: ImportManager): string {
...
 
+function genApiVersionDoc(apiVersions?: ApiVersions): string {
+  if (!apiVersions) {
+    return '';
+  }
+  const versions = new Array<string>();
+  apiVersions.forEach((val) => {
+    versions.push(val.version);
+  })
+  return `// Uses API version ${versions.join(',')}\n`;
```
Comment (RickWinter) [inline src/generator/operations.ts:598]: Is there defensive coding or are there really instances multiple versions are the basis for code gen?

Comment (jhendrixMSFT) [reply src/generator/operations.ts:598]: I asked @timotheeguerin about when we'd have more than one API version for an operation.

Comment (RickWinter) [reply src/generator/operations.ts:598]: nit, "//  Generated from API version ${versions.join(',')}, See {hyperlink to swagger}\n"

Might be a lot of redundant text

Comment (jhendrixMSFT) [reply src/generator/operations.ts:598]: I don't know that we have the swagger information available.

I can change the text as specified above if you think that's better.

**Review by chlowell — APPROVED** [review]

**Review by RickWinter — APPROVED** [review]

**Pull request merged**